webdriver/tests/bidi/browsing_context/load/load.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from webdriver.error import TimeoutException
+
+from tests.support.sync import AsyncPoll
+from .. import assert_navigation_info
```

Comment:
Hm, I have a hard time finding this assertion helper. Given the import path it should be [`bidi/browsing_context/__init__.py`](https://github.com/web-platform-tests/wpt/blob/224e0e2ca091e1a5d5673bdfcedf483927eb1252/webdriver/tests/bidi/browsing_context/__init__.py) but it's not included there. It's suspicious given that all the tests in CI are passing.

Reply:
Yeah, you're right 🤦🏻‍♀️ 
I've added it now